smart-contracts/osmosis/contracts/cl-vault/src/vault/deposit.rs

```diff
@@ -57,7 +56,7 @@ pub(crate) fn execute_any_deposit(
     }
 
     let twap_price = get_twap_price(deps.storage, &deps.querier, &env, 24u64)?;
-    let (token_in, out_denom, remainder, price) = if !deposit_info.base_refund.amount.is_zero() {
```

Comment:
nit: why does a pr about removing dex router change remainder here?

Reply:
Also not sure why we are changing this

Reply:
see long explanation below

Reply:
Makes sense now, to add to the confusion github's view cut this weirdly to make me think this was part of exact_deposit 😓